indico/modules/events/persons/templates/management/_person_list_row.html

```diff
@@ -26,7 +26,10 @@
     {% endif %}
     <td class="i-table name-column" data-searchable="{{ person.display_full_name|lower }}">
         {{ person.display_full_name }}
-        {% if person.is_untrusted %}
+        {% if person_data.roles['no_builtin_roles'] %}
+            <span class="icon-question"
+                  title="{%- trans %}This person currently has no ties to the event.{% endtrans -%}"></span>
+        {% elif person.is_untrusted %}
```

Comment:
This also sets the `untrusted` class on the `tr` which grays it out a bit. We should probably do the same for unused persons...

Reply:
Nevermind